### PR TITLE
[ServiceBus&EventHubs] fix mypy 0.931 errors

### PR DESCRIPTION
**Pull request by swathipil**
follow up to #22891

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
     from azure.core.credentials import TokenCredential
 
 _LOGGER = logging.getLogger(__name__)
-_Address = collections.namedtuple("Address", "hostname path")
+_Address = collections.namedtuple("Address", "hostname path")   # type: ignore
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py:49]: @yunhaoling - Can we change the inner named tuple to `_Address`?

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py:49]: yes, I think we could rename the tuple as long as this namedtuple is kept for internal usage.

Comment (swathipil) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py]: yep only used internally, not exposed to user.

**Pull request merged**